Resources/views/page_head_style.html.twig

```diff
@@ -5,6 +5,8 @@
     <link rel="stylesheet" type="text/css" href="{{ asset_url }}"/>
 {% endstylesheets %}
 
+<link rel="stylesheet" type="text/css" href="/bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css" />
```

Comment:
+1 on using `'bundles/ezplatformuiassets/vendors/flag-icon-css/css/flag-icon.min.css'`, this is the recommended way in the Symfony documentation: http://symfony.com/doc/current/cookbook/assetic/asset_management.html#fixing-css-paths-with-the-cssrewrite-filter

Side note: at some point, we should try to investigate that issue, it's really a pity to have such limitation...